src/components/StyleOption/StyleOption.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import './StyleOption.scss';
+
+function StyleOption(props) {
+  const initialStyle = props.initialStyle ? props.initialStyle : 'regular';
+  const [style, setStyle] = useState(initialStyle);
```

Comment:
I haven't checked out the code yet but I'm wondering if we need to have a state in this case. When the component calls `onStyleChange` this component should be rerendered with the props that have the latest `Style`, and then we can just use that `Style` to be the value of `<select />`.

Reply:
This is a good catch... I think Im just over-eager to use hooks.

Reply:
Removed state as was redundant.